CONTRIBUTING.md

```diff
@@ -1,73 +1,110 @@
+    costs.
+  * We do not do this automatically as the CI server is not sufficiently isolated and the performance results would
+    vary. Similarly, your personal laptop is likely to produce unreliable results.
+* If your PR is related to an existing issue, reference it in the PR's description (e.g., `fixes #123` or `refs #123`).
+* If you are not a member of the Hyrise organization, your PR will not be built by our CI server. Contact a maintainer
+  for this. They can add you to the organization or manually trigger builds from within Jenkins.
+* For your PR to be merged, it must pass the FullCI run. Set the FullCI tag in Github before committing to trigger the
+  complete (but more expensive!) run.
+* When merging your PR, copy your PR description (excluding the benchmark results) into the commit message. The commit
+  message of the squash merge should NOT contain the individual commit messages from your branch.
 
-- Testing the storage module: `./build/hyriseTest --gtest_filter="Storage*"`
-- Testing the table class: `./build/hyriseTest --gtest_filter="StorageTableTest*"`
-- Testing the RowCount test: `./build/hyriseTest --gtest_filter="StorageTableTest.RowCount"`
-
-## Performance Warnings
-- Sometimes, we have convenience functions, such as `ReferenceSegment::operator[]`, or workarounds, such as performing multiple stable sorts instead of a single one. Because these might negatively affect the performance, the user should be warned if a query causes one of these slow paths to be chosen. For this, we have the PerformanceWarning() macro defined in `performance_warning.hpp`.
-
-## Documentation
-- Most documentation should happen in the code or in the beginning of the header file
-- More complex documentation, such as an explanation of an algorithm that profits from images, can be put in the Wiki. Please make sure to link the Wiki page in the code - otherwise, no one will find it.
-
-# Review
-
-**Things to look for:**
-
-	- Guidelines (see above)
-	- Is the copy constructor deleted where it makes sense?
-	- Is the destructor virtual for base classes?
-	- Are unnecessary copies of heavy elements made? (prefer vector& over vector, but not int& over int)
-	- Did the author update documentation and dependencies (Wiki, README.md, DEPENDENCIES.md, Dockerfile, install_dependencies.sh, Jenkinsfile)
+## Reviewing PRs
+* Keep the whole picture in mind. Often, it makes sense to make two passes: One for the code style and line-level
+  modifications; one for checking how it fits into the overall picture.
+* Check if the PR sufficiently adds tests both for happy and unhappy cases.
+* Do not shy away from requesting changes on surrounding code that was not modified in the PR. Remember that after a PR,
+  the code base should be better than before.
+* Verify the CI results, including the change in procentual coverage, and check if the compile time or query performance
```

Comment:
procentual -> percentaged, percental

Reply:
Maybe just "how the reported coverage changed"?

Reply:
Works for me.